Cargo.toml

```diff
@@ -3,6 +3,22 @@ name = "rmathlib"
 description = "A Rust Port of R's C Library of Special Functions"
 version = "1.0.0"
 edition = "2021"
```

Comment:
```suggestion
edition = "2021"
documentation = "https://rmathlib.poweranalyses.org/rmathlib"
```

Reply:
Dunno, aren't we publishing this? Then the link would change to docs.rs/rmathlib in the near future.